src/main/steps/edge-case/subject-contact-details/content.ts

```diff
@@ -21,7 +21,13 @@ export const form: FormContent = {
       label: l => l.contactNumberLabel,
       hint: h => h.contactNumberHint,
 
-      validator: value => isFieldFilledIn(value) || isPhoneNoValid(value),
+      validator: value => {
+        if (isFieldFilledIn(value)) {
+          return isFieldFilledIn(value);
+        } else if (isPhoneNoValid(value)) {
+          return isPhoneNoValid(value);
```

Comment:
the validators already return the value so why not use the original approach?
`      validator: value => isFieldFilledIn(value) || isPhoneNoValid(value),`

Reply:
The isFieldFilledIn function already does a null check. It would be better to update the code in validation.ts than have the if / else if. Can we keep the validator the same as it was before (see below) and update the validation code instead
`validator: value => isFieldFilledIn(value) || isPhoneNoValid(value) `

Reply:
Thank you! Reverted to that just now!